schema/pool.go

```diff
@@ -0,0 +1,19 @@
+package schema
+
+import "github.com/gofrs/uuid"
+
+type (
+	JsonSchema struct {
+		ID  uuid.UUID `json:"-" faker:"uuid" db:"id" rw:"r"`
+		Url string    `json:"-" faker:"-" db:"url"`
```

Comment:
Rename to URL

Also I think it's safe to return the URL in json even if it's not really used anywhere.